installers-conda/build-environment.yml

```diff
@@ -2,13 +2,13 @@ name: spy-inst
 channels:
   - conda-forge
 dependencies:
-  - conda >=24.5.0
-  - conda-build >=24.5.0
-  - conda-lock >=2.5.7
-  - conda-standalone >=24.5.0
-  - constructor >=3.6.0
+  - conda =24.5.0
+  - conda-build =24.5.1
+  - conda-lock =2.5.7
+  - conda-standalone =24.5.0
+  - constructor =3.8.1
   - gitpython
   - mamba
-  - menuinst >=2.1.0
+  - menuinst =2.1.1
```

Comment:
I think it'd be a good idea to add a simple and dummy dependency here to check that the installers base env is updated between rc1 and rc2.

I know we're waiting for a new menuinst version to be released but since there's no clarity about it, I think that could be a good test for this PR.